testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -45,6 +45,15 @@
 IGNORE_NO_INFORMATION_WARNING = 'ignore:Found no information for attr:UserWarning'
 
 
+@pytest.fixture
+def dummy_universe_without_elements():
```

Comment:
Could you look at minimizing the number of warnings that gets issued by your new tests? I used 2 different approaches for that in #2886: filling all the required attributes in the fixture or filtering the warnings in the tests.

Reply:
3d7857d should have done the trick, only warnings left are parmed's ABCs from collections import.